frontend/packages/akr/src/redux/reducers/authorisation.ts

```diff
@@ -1,9 +1,14 @@
 import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 import { APIResponseStatus } from 'shared/enums';
 
-import { AddAuthorisationState, Authorisation } from 'interfaces/authorisation';
+import { Authorisation } from 'interfaces/authorisation';
 
-const initialState: AddAuthorisationState = {
+export interface AuthorisationState {
```

Comment:
Tätä mä jossain kohtaa itsekin mietin, että olisko nuo tilan hallintaan liittyvät interfacet hyvä olla täällä reducerejen ohessa. Ei ehkä ruveta muiden osalta tässä PR:ssä näitä kuitenkaan siirtämään.

Reply:
Jos niitä ei käytetä muualla niin voi ihan hyvin siirtää reducerin alle. Osa niistä ovat kuitenkin käytetty monessa eri paikassa.

Reply:
Mitä pikaisesti vilkaisin, niin selectorin alla on ekspliittisenä tyyppinä monesti määritelty state, mutta siinäpä se. Stateja ei itsessään käytetä muualta.

Statella itsellään saattaa olla tosin riippuvuuksia interfaceihin, esim.`ExaminationDateState`:lla interfaceen ?`ExaminationDates`.

Reply:
Selectoreissa ei oikeastaan stateja tarvita, kun ne tulee automaattisesti. Siirrän vielä niitä reducereihin.

Reply:
Tehty osittain.